state_machine/states/odlc.py

```diff
@@ -0,0 +1,29 @@
+"""Declares the ODLC state class."""
+from typing import Awaitable, Callable, ClassVar
+
+from state_machine.states.state import State
+
+
+class ODLC(State):
+    """
+    The ODLC state of the state machine.
+
+    This state represents the phase in which the drone is scanning the search area.
```

Comment:
Combine these two into one sentence, something like "This state represents the phase in which the drone is scanning the search area for ODLCs"